### DISCUSSION TITLE
deploykf-gateway failed image pull issue

### DISCUSSION THREAD

**Discussion by tppalani** [deployKF/deployKF#209] [Questions & Answers] [answered]
I'm getting below image pull error when i' doing sync the application dkf-core--deploykf-istio-gateway 

```
Failed to pull image "auto": rpc error: code = Unknown desc = failed to pull and unpack image "docker.io/library/auto:latest": failed to resolve reference "docker.io/library/auto:latest": pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed
```

**Comment by thesuperzapper** [comment, marked as answer]
@tppalani Your istio-gateway pods would only be trying to pull the `auto` image if you had not installed Istio correctly, as Istio automatically replaces the `auto` image with the correct one if its installed.

Please follow the [getting started guide](https://www.deploykf.org/guides/getting-started/), and ensure you are syncing with the sync script, rather than manually.

Also, I recommend playing around with the [local quick start](https://www.deploykf.org/guides/local-quickstart/) on a laptop if you have an X86 machine, or a cloud instance if you have a MacBook with ARM chip (ARM is not supported yet) to better understand how stuff works.
